docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -289,7 +285,7 @@ The geometry section determines where each motor is on the aircraft and what dir
 
     Motor to Actuator to CVI Associations
 
-If you accidentally put the wrong CVI on a motor you can just switch the motor functions in the actuator outputs section. If for example you accidentally set the front right motor as CVI 2 and the front left motor as CVI 0, then you can just set ESC 1 as Motor 3, and ESC 3 as Motor 1. You could also change the CVIs associated with each module, but we find that this is the most straightforward way to swap connections.
+If you accidentally put the wrong CVI on a motor you can just switch the motor functions in the actuator outputs section. If for example you accidentally set the front right motor (Module ID 0) as CVI 2 and the front left motor (Module ID 2) as CVI 0, then you can just set ESC 1 as Motor 3, and ESC 3 as Motor 1. You could also change the CVIs associated with each module, but we find that this is the most straightforward way to swap connections.
```

Comment:
you left in some old ESC naming

Reply:
Good catch. Fixed with [db7f75e](https://github.com/iq-motion-control/iq-module-communication-doc/pull/103/commits/db7f75e1e4bcc5e5e47b5372253ec3d035181dee)